salt/metalk8s/addons/logging/fluent-bit/deployed/configmap.sls

```diff
@@ -109,7 +113,9 @@ Create fluent-bit ConfigMap:
             }
           parsers.conf: |-
             [PARSER]
-                Name        docker
-                Format      json
+                Name        container
+                Format      regex
+                Regex       ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]+) (?<message>.+)$
```

Comment:
I think this parser should always be used for `containerd`-formatted logs, no? And as such, likely this commit should be backported to the first version where Loki was introduced (2.6, I guess?)

Reply:
Yes, I agree, we should probably backport it.